### DISCUSSION TITLE
Request housekeeping for https://app.dework.xyz/darwinia-dao

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#98] [General]
We have not been using dework for a long time, is it still a useful tool for us?

Just found some new features there.

**Comment by hackfisher** [comment]
And https://guild.xyz/darwinia

**Comment by 2xJDubs** [comment]
We have been using it for the Ambassadors and now the one DCDAO Advocate.  They could easily migrate to posting their monthly reports on github as well.
